src/java/org/apache/cassandra/db/compression/CompressionDictionaryManager.java

```diff
@@ -244,6 +251,68 @@ public CompositeData getTrainingState()
         return dictionaryTrainer.getTrainingState().toCompositeData();
     }
 
+    @Override
+    public TabularData listDictionaries(String keyspace, String table)
+    {
+        List<LightweightCompressionDictionary> dictionaries = SystemDistributedKeyspace.retrieveLightweightCompressionDictionaries(keyspace, table);
```

Comment:
We should use the field variables of the enclosing `CompressionDictionaryManager` instance. 
```
    private final String keyspaceName;
    private final String tableName;
```